public/app/features/alerting/unified/components/notification-policies/Policy.tsx

```diff
@@ -513,9 +513,7 @@ function MetadataRow({
         )}
         {timingOptions && (
           // for the default policy we will also merge the default timings, that way a user can observe what the timing options would be
-          <TimingOptionsMeta
-            timingOptions={isDefaultPolicy ? defaults(timingOptions, TIMING_OPTIONS_DEFAULTS) : timingOptions}
-          />
+          <TimingOptionsMeta timingOptions={timingOptions} />
```

Comment:
Had to remove applying defaults here, it was only applying the default timings cosmetically.